lib/env.js

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('./config');
+
+exports.isProdLike = function isProdLike() {
+  var env = config.get('env');
+  return env === 'prod' || env === 'stage';
```

Comment:
Optionally, another route is to move this requirement away from `isProdLike`, and instead add a `config.events.required` value, default to true, and if `env` is `dev` or `test`, switch to false. Then, latest could also just set this requirement to false... :fried_shrimp: